WcaOnRails/config/locales/en.yml

```diff
@@ -330,6 +333,9 @@ en:
         showAtAll: ""
         start_date: ""
         use_wca_registration: ""
+        competitor_limit_flag: "This is currently for information purposes only and will not affect the registration flow. We are working on adding explicit support for this to the registration flow, but it requires some other work first."
```

Comment:
"For now this number is informational only, and does not yet prevent more people from registering."